.github/workflows/daily.yml

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
`daily.yml` sounds a bit confusing to me

Reply:
I didn't come up with better name

Reply:
@birtony Do you have any suggestions? Were you thinking of something more descriptive, perhaps `dailybuild.yml`?

Reply:
Something like that, @DRK3